orchestrator_service/src/main/java/com/override/orchestrator_service/model/Keyword.java

```diff
@@ -24,7 +24,7 @@ public class Keyword {
     @Column(name = "last_used")
     private LocalDateTime lastUsed;
 
-    @Column(name = "usage_count")
+    @Column(name = "usage_count", columnDefinition = "INTEGER DEFAULT 1")
     private Integer usageCount;
```

Comment:
вероятно тут просто спасет 
```suggestion
    private Integer usageCount = 1;
```